config/manifests/bases/mongodb-atlas-kubernetes.clusterserviceversion.yaml

```diff
@@ -60,101 +60,96 @@ spec:
       name: atlasteams.atlas.mongodb.com
       version: v1
   description: |
-    The MongoDB Atlas Operator provides a native integration between the Kubernetes orchestration platform and MongoDB Atlas —
-    the only multi-cloud document database service that gives you the versatility you need to build sophisticated and resilient applications that can adapt to changing customer demands and market trends.
-
-    > Current Status: *Stable*. The Operator gives users the ability to provision
-    > Atlas projects, clusters and database users using Kubernetes Specifications and bind connection information
-    > into applications deployed to Kubernetes.
+    The MongoDB Atlas Operator provides a native integration between the Kubernetes orchestration platform and MongoDB Atlas
+    — the only multi-cloud document database service that gives you the versatility you need to build sophisticated and
+    resilient applications that can adapt to changing customer demands and market trends.
```

Comment:
@helderjs please take this text in a consumable form and let it ratify by @dan-mckean

Reply:
@s-urbaniak this is a copy & paste from README, exactly the same content.